src/components/Body/MovieCrads/MovieCard/MovieCard.tsx

```diff
@@ -11,18 +19,43 @@ type MoviePropsType = {
 };
 
 function MovieCard({ movie }: MoviePropsType) {
+  const options = useAppSelector(selectFilterBy);
+  const searchQuery = useAppSelector(selectSearchQuery);
+  const sortBy = useAppSelector(selectSortBy);
   const dispatch = useAppDispatch();
```

Comment:
Query params can be used to store state as well (at least, in our use case). Now it turned out, that state is stored in 2 places. That is a bad practice, because we have to manage 2 stores instead of 1 and we have to keep them in sync. It will be easier to use router instead of redux, where possible.